### DISCUSSION TITLE
PWM output

### DISCUSSION THREAD

**Discussion by bseishen** [ExpressLRS/ExpressLRS#536] [Ideas]
PWM output for planes without an FC to drive esc's and servos. Be handy to have, and part of the frsky r9 feature set.

**Comment by MarkusPfaff** [comment]
Yes, express LRS would be very interesting for normal RC receivers for fixed wing models which have PWM outputs for servos.

**Comment by greenbigfrog** [comment]
https://github.com/ExpressLRS/ExpressLRS/issues/581

**Comment by bwoostdam** [comment]
PWM support has been established however it is restricted to 4 channels at the moment. I think that support for at least 1 more full range PWM channel is very very much needed as many many pilots have planes with 2 aileron servo's and would like to connect them to the RX separately.
Really hope this will come soon.

**Comment by robhaswell** [comment]
Matek make a 7ch PWM RX now. So can this be closed?

**Comment by bwoostdam** [comment]
Yes thank you. It can be closed. Will look into it!!
